woocommerce/woocommerce-paypal-payments
# Fix/pcp 4034 update pricing badges

### Description

This PR updates the pricing badges with new values. It also adds Fastlane badge and Pay Later. 

### Steps to Test

1. Enable the new UI settings
2. Go to PayPal settings page
3. The settings show the badges with updated values
4. Change the country store one of the countries in the table and the currency to one of the values in the table and see the changes reflected in the PayPal settings page


Country | Currency Symbol | Fixed Fee | PayPal Checkout | Pay Later | Custom Card Fields / Digital Wallets / Fastlane | Alternative Payment Methods | Standard Card Fields | Fees Page
-- | -- | -- | -- | -- | -- | -- | -- | --
United States | $ (USD) | 0.49 USD0.39 GBP0.59 CAD0.59 AUD0.39 EUR | 3.49% | 4.99% | 2.89% + $0.29 | 2.89% + $0.29 | 2.99% | https://www.paypal.com/us/webapps/mpp/merchant-fees
United Kingdom | £ (GBP) | 0.30 USD0.30 GBP0.30 CAD0.30 AUD0.35 EUR | 2.90% | 2.90% | 1.20% | 1.20% | 1.20% | https://www.paypal.com/uk/webapps/mpp/merchant-fees
Canada | $ (CAD) | 0.30 USD0.20 GBP0.30 CAD0.30 AUD0.35 EUR | 2.90% | N/A | 2.70% | 2.90% | 2.90% | https://www.paypal.com/ca/webapps/mpp/merchant-fees
Australia | $ (AUD) | 0.30 USD0.20 GBP0.30 CAD0.30 AUD0.35 EUR | 2.60% | 2.60% | 1.75% | 2.60% | 2.60% | https://www.paypal.com/au/webapps/mpp/merchant-fees
France | € (EUR) | 0.30 USD0.30 GBP0.30 CAD0.30 AUD0.35 EUR | 2.90% | 2.90% | 1.20% | 1.20% | 1.20% | https://www.paypal.com/fr/webapps/mpp/merchant-fees
Italy | € (EUR) | 0.30 USD0.30 GBP0.30 CAD0.30 AUD0.35 EUR | 3.40% | 3.40% | 1.20% | 1.20% | 1.20% | https://www.paypal.com/it/webapps/mpp/merchant-fees
Germany | € (EUR) | 0.49 USD0.29 GBP0.59 CAD0.59 AUD0.39 EUR | 2.99% | 2.99% | 2.99% | 2.99% | 2.99% | https://www.paypal.com/de/webapps/mpp/merchant-fees
Spain | € (EUR) | 0.30 USD0.30 GBP0.30 CAD0.30 AUD0.35 EUR | 2.90% | 2.90% | 1.20% | 1.20% | 1.20% | https://www.paypal.com/es/webapps/mpp/merchant-fees
